test/e2e/install/bla.go

```diff
@@ -0,0 +1,25 @@
+package install
```

Comment:
`bla.go`?

Reply:
LOL, will fix it in a minute

Reply:
OK the file has been deleted in a later commit, but just to be extra righteous I'll clean the history